.travis.yml

```diff
@@ -12,10 +12,12 @@ env:
   - TRAVIS_GOOS=windows
   - TRAVIS_GOOS=linux
   - TRAVIS_GOOS=darwin
+  - TRAVIS_GOOS=solaris
 
 script:
 # TODO: vendor
   - export GOOS=${TRAVIS_GOOS}
   - go get -t ./...
   - make build binaries
   - if [ "$GOOS" = "linux" ]; then make vet test; fi
+  - if [ "$GOOS" = "solaris" ]; then go test -c; fi
```

Comment:
turns out 'go test -c ...' doesn't allow you to specify multiple packages. so now i'm running 'go test -c <pkg>' for each package returned by 'go list ./...'